dynamics:
  mass_car: 1.0
  tau_theta: 0.7
  substep_size: 0.001



solver:
  rollouts: 200
  lambda: 0.0 #0.001
  h: 20.0
  step_size: 0.01
  horizon: 3.0

  cost_ratio: 0.2
  caching_factor: 0.1
  filtering: false
  substeps: 1

  filter_type: 0
  filter_order: 2
  filter_window: 10

  adaptive_sampling: false
  input_variance: [1.0]

  discount_factor: 1.0
  verbose: false
  threads: 1  # it works only with one thread ... investigate this further
  debug_print: false

  use_tree_search: false
  pruning_threshold: 0.01      # check with 9999999
  #  expert_types: {NORM, IMP}
  expert_weights: [ 0.0, 1.0 ]
  log_data: true
  bound_input: false
  u_min: [ -1.0 ]
  u_max: [ 1.0]

policy_update_rate: 0.0  # execute as fast as possible
reference_update_rate: 10.0
publish_ros: true
ros_publish_rate: 10.0

static_optimization: false
sim_dt: 0.01